biz.aQute.bnd.exporters/src/aQute/bnd/exporter/feature/json/FeatureJsonHandler.java

```diff
@@ -0,0 +1,129 @@
+package aQute.bnd.exporter.feature.json;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.osgi.util.feature.Feature;
+import org.osgi.util.feature.FeatureConfiguration;
+import org.osgi.util.feature.ID;
+
+import aQute.lib.json.Encoder;
+import aQute.lib.json.Handler;
+import aQute.lib.json.StringHandler;
+
+public class FeatureJsonHandler extends Handler {
```

Comment:
I really do not like the idea of having specific handlers ... it mixes concerns. You can easily build up the JSON outside by creating appropriate DTOs. This kind of design tends to make more and more complexity over time.

Reply:
I started with the dto way but i will need the ability to write comment's into the json. See last example here.https://github.com/bndtools/bnd/issues/4586

And i have jo ideas how to do it with dtos.